<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sub_model</class>
 <widget class="QWidget" name="sub_model">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="Box_attribute">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>20</y>
     <width>120</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>属性</string>
   </property>
   <widget class="QCheckBox" name="check_facial">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>表情</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_gender">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>性别</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_age">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>年龄</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_upload">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>上传云端</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>0</y>
     <width>241</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="Box_cotrol">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>280</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>控制</string>
   </property>
   <widget class="QPushButton" name="pub_rs">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>R/S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pub_cleartext">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>文本清空</string>
    </property>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="Graphic_main">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>541</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
